src/_posts/whats-new-in-0.16.0.md

```diff
@@ -0,0 +1,128 @@
+
+<p class="tip">Note that it is extremely important that you ensure the test suite is passing when using this option, otherwise test failures would appear like caught mutations and those mutations may be reported incorrectly.</p>
+
+### Other performance improvements
+
+TBD
+
+* https://github.com/infection/infection/pull/1177
+* https://github.com/infection/infection/pull/1172
+* https://github.com/infection/infection/pull/1106
+* https://github.com/infection/infection/pull/1082
+
+## Splitting Infection by separate Packages
+
+Infection has been split by different packages. For example, we have extracted `PhpSpec` and `Codeception` adapter out of the core.
+
+You can read an very detailed explanation about why was it needed in [this RFC](https://github.com/infection/infection/issues/922).
+
+In a nutshell, this is useful both for the end user and for maintainers:
+
+* you will not download not needed code if you don't use those test frameworks
+* you will not need to upgrade Infection if for example bug is fixed in the test framework you don't use
+* now Test Frameworks have clear dependencies and requirements
+
+Don't worry, if you are using Codeception or PhpSpec, these packages will be automatically installed at the first Infection execution after upgrading to 0.16.0.
+
+List of new packages:
+
+* [`infection/codeception-adapter`](https://github.com/infection/codeception-adapter)
+* [`infection/phpspec-adapter`](https://github.com/infection/phpspec-adapter)
+* [`infection/abstract-testframework-adapter`](https://github.com/infection/abstract-testframework-adapter) - interfaces for any testframework adapter
+* [`infection/extension-installer`](https://github.com/infection/extension-installer) - autodiscovery of installed adapters. No need to add them to `infection.json`, just install a Composer package and you are done. Inspired by PHPStan and Psalm plugin system.
+* [`infection/include-interceptor`](https://github.com/infection/include-interceptor) - Stream Wrapper. Allows to replace included (autoloaded) file with another one (magic that replaces your code with a mutated one) 
+
+
+## Mutation Badge
+
+We had this feature [before](/guide/mutation-badge.html), but it worked only for TravisCI.
+
+Now, thanks to [`ondram/ci-detector`](https://github.com/OndraM/ci-detector), Mutation Badge can be used for [all the supported](https://github.com/OndraM/ci-detector#supported-continuous-integration-servers) by this lib CI servers, including GitHub Actions, CircleCI, GitLab and so on.
+
+Feel free to request an additional integration there if the list of currently supported continuous-integration servers does not contain a service you require.
+
+> Read more about how to setup [Mutation Badge](/guide/mutation-badge.html)
+
+## Global ignore feature for Mutators and Profiles
```

Comment:
@theofidry feel free to add anything you think should be in this post as well, as there were tons of PRs from you, unfortunately without `0.16.0` milestone. I've added those that found there